packages/contentful--cra-template-create-contentful-app/template.json

```diff
@@ -5,7 +5,7 @@
       "@contentful/f36-components": "^4.0.0",
       "@contentful/f36-tokens": "^4.0.0",
       "contentful-management": "^7.45.0",
-      "emotion": "^10.0.17",
```

Comment:
Why not the latest version?

Reply:
This was just the one that dependabot was trying to get to.